app.rb

```diff
@@ -1,24 +1,30 @@
 # frozen_string_literal: true
 
 require 'sequel'
+require 'sensible_logging'
 require 'sinatra/base'
 require 'sinatra/json'
 require './lib/loader'
 
 class App < Sinatra::Base
+  register Sinatra::SensibleLogging
+
   configure do
-    enable :logging
     enable :json
 
-    set :logging, Logger::DEBUG
+    sensible_logging(
```

Comment:
This should be moved outside of the `configure` block. Then we can just configure `log_level` for the various environments.